Gruntfile.js

```diff
@@ -48,9 +48,7 @@ module.exports = function(grunt) {
           'test/lib/unittests-bundle.js': [ './test/unittests.js' ]
         },
         options: {
-          browserifyOptions: {
-            external: [ 'openpgp', 'crypto', 'node-localstorage']
-          }
+          external: ['crypto', 'node-localstorage', 'openpgp', '../../../dist/openpgp', '../../dist/openpgp']
```

Comment:
@evilaliv3 to answer your question... this is how things should be excluded just to be sure the browserified unittest.js build does not contain any src files in any of the unit tests. E.g. `test/crypto/cipher/aes.js` is one level deeper than `test/crypto/crypto.js`.